### ISSUE TITLE
libertatea.net

### ISSUE THREAD

**Issue by BottledSoda** [uBlockOrigin/uAssets#11518]
### Prerequisites

- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues) to avoid opening a duplicate issue
- [X] I am running the latest version of uBlock Origin

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### URL(s) where the issue occurs

`libertatea.net`

### Describe the issue

ultimately got redirected from it to a link which was blocked by uBO

### uBlock Origin version

1.40.8

### Browser name and version

Google Chrome

### Settings

.

**Comment by BottledSoda** [comment]
I wanted to visit an official `Libertatea` site.

**Comment by mapx-** [comment]
`libertatea.net` is dead / parked

**Comment by BottledSoda** [comment]
> `libertatea.net` is dead / parked

that's seemingly a common thing for such domains, do u want a screenshot of a part of my browsing history ? also, did you try with multiple IPs ?

**Comment by mapx-** [comment]
The site is parked, sometimes these domains redirecting to some crap.
The trip is ending on some blocked domain. That's all.

**Comment by BottledSoda** [comment]
> The site is parked, sometimes these domains redirecting to some crap. The trip is ending on some blocked domain. That's all.

Okay, but `libertatea.net^$doc` would still be good for people wanting to visit the official website of the `Libertatea TV` (or whatever it's called) news station's official website for news.

**Comment by BottledSoda** [comment]
Also, what if the redirect isn't to a blocked domain? What if `||0redirc.com^$doc` wasn't in `uBlock filters -- Badware risk` ?
